_foundation/integration-onboarding.md

```
@@ -0,0 +1,766 @@
+    - [2a. Management Portal authentication (Microsoft Entra ID)](#2a-management-portal-authentication-microsoft-entra-id)
+    - [2b. Customer Portal authentication via Microsoft Entra ID](#2b-customer-portal-authentication-via-microsoft-entra-id)
+    - [2c. Customer Portal authentication via MitID](#2c-customer-portal-authentication-via-mitid)
+5. [Entra SCIM provisioning](#3-entra-scim-provisioning)
+6. [Engage Public API](#4-engage-public-api)
+7. [Dynamics 365 CRM integration](#5-dynamics-365-crm-integration)
+
+---
+
+## Starting inputs
+
+A few decisions and items apply across multiple sections of this document. They are resolved before per-section work begins and are referenced by the [Prerequisites](#prerequisites) tables in each section.
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)